scripts/abi_artifact_generator/config/readme.md

```diff
@@ -0,0 +1,99 @@
+# Documentation for the configuration options provided by this generator
```

Comment:
Please start off stating the goal of the tool and how to invoke it

Reply:
Something like this?
```markdown
The goal of this generator is to have a combined interface for generating test for each of the testeed architectures by taking advantage of multiple templates with varying levels of genericness.

This file explains the said configuration in multiple stages:
* [the values shared between all the tests](#commonyml)
* [the way to specify the list of functions for the tests to be based on](#functionsyml)
* [the way to add/modify a specific architecture to the testing suite](#architecturesyml)
```

Reply:
On it.

Reply:
Closing this, since there are new related threads.